spec/rubocop/cop/sketchup_suggestions/command_title_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::SketchupSuggestions::CommandTitle do
```

Comment:
To make this pass the tests we need to change this line to:

```
describe RuboCop::Cop::SketchupSuggestions::CommandTitle, :config do
```

And then remove:

```
subject(:cop) { described_class.new }
```

Reply:
That is what was done to all our existing cops to fix the build:
https://github.com/SketchUp/rubocop-sketchup/commit/679d2e2794eff93e4c4c8251ee8bdd9426ce4b60